README.md

```diff
@@ -1,7 +1,62 @@
-# TeamFourHYF
-Recreation of HYF website
+# Project Definition
 
+## About
+
+With related to our course **Incremental Development** module and in practicing of **Planning & Collaboration on software projects**,we here by tried to **Recreate our school's** [website](https://hackyourfuture.be/).
+
+## Technologies
+
+The following tools were used in this project:
+
+- VScode
+- GitHub
+- Git
+- Html
+- CSS
+- Slack
+
+## General Steps
+
+  1. [Create and Setup GitHub Repository](https://github.com/lauraramirez220612/TeamFourHYF)
+  2. [Write Project Definition](README.md)
+  3. [Write Backlog](planning/backlog.md)
+  4. [Make Wire-frame](planning/wireframe.md)
+  5. [Development Strategy](planning/development-strategy.md)
+  6. [Development](https://github.com/lauraramirez220612/TeamFourHYF/blob/main/index.html)
+
+## Screenshots
+
+Intended final result
+
+- [Screenshot image 1](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165146_hackyourfuture.be.jpeg)
+- [Screenshot image 2](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165220_hackyourfuture.be.jpeg)
+- [Screenshot image 3](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165247_hackyourfuture.be.jpeg)
+
+- [Screenshot image 4](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165345_hackyourfuture.be.jpeg)
+
+- [screenshot image 5](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165443_hackyourfuture.be.jpeg)
+
+- [Screenshot image 6](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165512_hackyourfuture.be.jpeg)
+- [Screenshot image 7](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165538_docs.google.com.jpeg)
+
+## Contributors
+
+- [Bringmetheaxx](https://github.com/bringmetheaxx)
+
+- [Daniel Halasz](https://github.com/danielhalasz)
+
+- [Moamin-AbuEwaida](https://github.com/Moamin-AbuEwaida)
+
+- [Laura-Ramirez](https://github.com/lauraramirez220612)
+
+- [TatsianaRud](https://github.com/TatsianaRud)
+
+- [Senait Berhane](https://github.com/Senait-coding)
+
+## Coach
+
+[Unmesh](https://github.com/unmeshvrije)
```

Comment:
i think it would be nice to add stephane as well

Reply:
definitely

Reply:
ok  I will add him, thanks

Reply:
@Moamin-AbuEwaida could you give me the link of Stéphane GitHub link please,  Unmesh is included in our repo so, I found his GitHub link  easily but couldn't find Stephane's  or else I  will check  in our previous  repo

Reply:
https://github.com/snicoll

Reply:
Thank you  @snicoll, I  already found your links from our previous Repo when Moamin got late to replay me :) and  I have amended the Readme accordingly.